packages/legacy/core/App/components/modals/CommonRemoveModal.tsx

```diff
@@ -75,7 +75,7 @@ const CommonRemoveModal: React.FC<CommonRemoveModalProps> = ({ removeType, visib
     controlsContainer: {
       marginTop: 'auto',
       marginHorizontal: 20,
-      marginBottom: 108,
+      marginBottom: Platform.OS === 'ios' ? 108 : 20,
```

Comment:
I'm curios what's going on here, does React-Native naturally render the margins differently on Android and IOS, and then this change compensates for that?

Reply:
Its because the big gap that android adds to the title (no idea why). Here is how it looks without that style:
![unnamed (1)](https://user-images.githubusercontent.com/112953503/234386862-8a867335-5f54-4fef-b7db-fd3f1cbed69d.png)